index.html

```diff
@@ -854,18 +854,14 @@ <h2 id="docconformance">
               <a>No corresponding role</a>
             </td>
             <td>
-              <p>
-                Roles:
-                <a href="#index-aria-application">`application`</a>,
-                <a href="#index-aria-document">`document`</a>,
```

Comment:
What could possibly be wrong with these?

Reply:
no positive benefit so far.  can break access to content.